lib/github_fetcher.rb

```diff
@@ -128,4 +127,24 @@ def explicitly_included_repo?(repo)
   def repo_name(pr)
     pr.html_url.split("/")[4]
   end
+
+  def get_team_members(team_slug, members)
+    github_team_members = get_github_team_members(team_slug)
```

Comment:
I'd move the presence check to this method, so that it uses the `team_slug` parameter rather than just passing it to `get_github_team_members`. Also not sure under what circumstances `team_slug` would evaluate to an empty string rather than `nil`. Best to leave that bit out unless we know we need it.

```suggestion
    return members if team_slug.nil?

    github_team_members = get_github_team_members(team_slug)
```